SkatkovAM/lab1.cpp

```diff
@@ -0,0 +1,79 @@
+#include <iostream>
+#include <limits>
+using namespace std;
+float a,b;
+int onOffInd,inputInd=1; //ind:0=right input; 1= wrong input; on
+char operation;
+void hello(){
+    cout<<"Hello. im calculator ^_^"<<endl;
+}
+
+void onOffCheck(){
+    if(a==0 && b==0 && operation=='0'){
+        onOffInd=1;
```

Comment:
Functions should communicate with each other through return values, not global variables. If you decided to use function - consider rewriting them and eliminating all global variables

Reply:
Is it about all functions, or only  simple like that one?